draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1110,25 +1129,17 @@ whose output distribution is not uniformly random, and random oracle encodings,
 whose output distribution is indistinguishable from uniformly random.
 Some protocols require a random oracle for security, while others can
 be securely instantiated with a nonuniform encoding.
-When the required encoding is not clear, applications SHOULD use a random oracle.
 
 Care is required when constructing a random oracle from a mapping function.
 A simple but insecure approach is to use the output of a cryptographically
-secure hash function H as the input to the mapping.
+secure hash function as the input to the mapping.
 Because in general the mapping is not surjective, the output of this
 construction is distinguishable from uniformly random, i.e., it does
 not behave like a random oracle.
 
-Brier et al. {{BCIMRT10}} describe two generic methods for constructing
-random oracle encodings.
-Farashahi et al. {{FFSTV13}} and Tibouchi and Kim {{TK17}} refine the analysis
-of one of these constructions.
-That construction is described in {{roadmap}}.
-
-(In more detail: both constructions are
-indifferentiable from a random oracle {{MRH04}} when instantiated
-with appropriate hash functions modeled as random oracles.
-See {{security-considerations}} for further discussion.)
+The construction described in {{roadmap}} is indifferentiable from a random
```

Comment:
👍

Reply:
We should also give credit to {{BCIMRT10}} as it was proposed earlier.